ext/datadog_profiling_native_extension/ruby_helpers.h

```diff
@@ -115,3 +115,5 @@ size_t ruby_obj_memsize_of(VALUE obj);
 // return a string with the result of that call. Elsif the object responds to
 // 'to_s', return a string with the result of that call. Otherwise, return Qnil.
 VALUE ruby_safe_inspect(VALUE obj);
+
+VALUE ddtrace_version(void);
```

Comment:
Minor: I guess we could rename this `datadog_version` (and the places where it gets referenced from), but it's pretty small so OK if we leave if for later (this is not a public API anyway)